react-frontend/src/components/Navigation/NavLink/NavLink.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useLocation } from 'react-router-dom'
+import './NavLink.css'
+import PropTypes from 'prop-types'
+
+const NavLink = ({ href, label }) => {
+  const location = useLocation()
+  const isActive = location.pathname === href
+
+  return (
+    <a href={href} className={`nav-link ${isActive ? 'active' : ''}`}>
```

Comment:
We can use `classnames` https://www.npmjs.com/package/classnames instead

Reply:
Thanks for the idea. Refactored in [0c2832f](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/0c2832f95bc0ca4eb2cb355a368e619b1378eb2b)